### DISCUSSION TITLE
Don't you have a calendar tool?

### DISCUSSION THREAD

**Discussion by davidjayjackson** [DaxStudio/DaxStudio#1143] [General]
Say, don't you have a tool for creating date tables? TX David

**Comment by marcosqlbi** [comment]
You can use Bravo for Power BI: https://bravo.bi
It has the Manage Dates feature: https://docs.sqlbi.com/bravo/features/manage-dates/
Which uses the DAX Template library: https://docs.sqlbi.com/dax-template/
DAX Studio does not manipulate the data model. The library could be integrated, but it would change the nature of DAX Studio.

**Comment by davidjayjackson** [comment]
Thanks, that was what I was looking for.
David
